org.mixedrealitytoolkit.uxcore/Button/PressableButton.cs

```diff
@@ -462,7 +510,10 @@ protected override void OnHoverEntered(HoverEnterEventArgs args)
                 }
 
                 validPokeInteractors.Add(pokeInteractor);
+                isHovered = true;
```

Comment:
We should be reusing the base `IsHovered` value

Reply:
I missed that, thanks for catching it and letting me know.  I removed it in the latest commit.  Resolving this.